php/libraries/LorisForm.class.inc

```diff
@@ -1352,11 +1352,11 @@ class LorisForm
      *
      * @param callable $func The callback to use to process this form.
      *
-     * @return none
+     * @return mixed Preferably `bool`...
```

Comment:
If it's expecting a bool, I think this should just be documented to return a bool, so that phan can catch anywhere that doesn't return the proper type.